src/tests/NXTTest.h

```diff
@@ -130,6 +130,19 @@ class NXTTest : public ::testing::TestWithParam<BackendType> {
         utils::BackendBinding* binding = nullptr;
 };
 
+class DrawQuad {
+    public:
+        DrawQuad() {}
+        DrawQuad(nxt::Device* device, const char* vsSource, const char* fsSource);
```

Comment:
DrawQuad seems like it isn't doing enough for now, we should be able to ask where the quad should be drawn, and only pass the fs for example.